### PR TITLE
Ribn 563

### PR DESCRIPTION
**Pull request by ultimaterex**
Targets #182 as well

### CODE REVIEW

File: lib/presentation/asset_details/asset_detail_items/asset_icon_details.dart [hunk 1]
```diff
@@ -36,8 +36,8 @@ class AssetIconDetails extends StatelessWidget {
           height: 30,
           child: Row(
             children: [
-              const Text('Icon', style: RibnToolkitTextStyles.h4),
-              const Spacer(),
+              // const Text('Icon', style: RibnToolkitTextStyles.h4),
+              // const Spacer(),
```
Comment (KalervoHyyppa) [inline lib/presentation/asset_details/asset_detail_items/asset_icon_details.dart:40]: Is this widget even used? From what I'm seeing it isn't

File: lib/presentation/asset_details/asset_details_page.dart [hunk 2]
```diff
@@ -196,6 +166,11 @@ class _AssetDetailsPageState extends State<AssetDetailsPage> with RouteAware {
                         _buildDivider(),
                         // asset code display
                         AssetCodeDetails(assetCode: widget.assetCode),
+                        _buildDivider(),
+                        //TODO: Figure out how to pipe metadata in
```
Comment (KalervoHyyppa) [inline lib/presentation/asset_details/asset_details_page.dart:170]: Is there a ticket for this TODO?

File: lib/presentation/asset_details/asset_details_page.dart [hunk 3]
```diff
@@ -147,46 +142,21 @@ class _AssetDetailsPageState extends State<AssetDetailsPage> with RouteAware {
...
-                        AssetIconDetails(
-                          key: assetIconKey,
-                          currIcon: assetDetails?.icon,
-                          editingSectionOpened: editingAssetIcon,
-                          onEditPressed: () => _onEditPressed(
-                            key: assetIconKey,
-                            assetDetails: assetDetails,
-                          ),
+                        // asset name display
+                        Row(
```
Comment (KalervoHyyppa) [inline lib/presentation/asset_details/asset_details_page.dart:148]: Would making the spacing even for both these row items match the Figma design? You could just wrap both with an 
```
Expanded(
   flex: 1,
   child: ....
)
```

**Review by KalervoHyyppa — CHANGES_REQUESTED** [review]
See comments

File: lib/presentation/asset_details/asset_details_page.dart (hunk 2 above)
Comment (ultimaterex) [reply lib/presentation/asset_details/asset_details_page.dart:170]: @KalervoHyyppa There is not yet no

File: lib/presentation/asset_details/asset_detail_items/asset_icon_details.dart (hunk 1 above)
Comment (ultimaterex) [reply lib/presentation/asset_details/asset_detail_items/asset_icon_details.dart:40]: @KalervoHyyppa not anymore no, updating pr with this removed

File: lib/presentation/asset_details/asset_details_page.dart (hunk 3 above)
Comment (ultimaterex) [reply lib/presentation/asset_details/asset_details_page.dart:148]: resolved

**Review by KalervoHyyppa — APPROVED** [review]
LGTM

**Pull request merged**